pkg/controller/bundle/bundle_unpacker_test.go

```diff
@@ -398,18 +399,18 @@ func TestConfigMapUnpacker(t *testing.T) {
 			},
 		},
 		{
-			description: "CatalogSourcePresent/ConfigMapPresent/JobPresent/Unpacked",
+			description: "CatalogSourcePresent/ConfigMapPresent/JobPresent/DigestImage/Unpacked",
 			fields: fields{
 				objs: []runtime.Object{
 					&batchv1.Job{
 						ObjectMeta: metav1.ObjectMeta{
-							Name:      pathHash,
+							Name:      hash(digestPath),
```

Comment:
Rather than calling `hash(digestPath)` multiple times, could we define `digestHash` once upfront like we did for `pathHast`

Reply:
Yes good call out, updated